pkg/api/v1/types.go

```diff
@@ -545,15 +545,15 @@ type RBDVolumeSource struct {
 	// The rados pool name.
 	// Default is rbd.
```

Comment:
These comments are incorrect. "rbd" is not a default value AFAICT.

Default values are set explicitly:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#defaulting

If they were set, it should be here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/defaults.go

We can't change this in v1 without breaking backward compatibility. Please file an issue to fix this.

The terminology I suggested elsewhere is "Implicitly inferred to be rbd if unset."